Sources/SwiftSyntax/LexerError.swift

```diff
@@ -46,6 +45,7 @@ public struct LexerError: Hashable {
   public init(_ kind: Kind, byteOffset: Int) {
     assert(byteOffset >= 0)
     // `type(of: self.byteOffset).max` gets optimized to a constant
+    assert(byteOffset >= 0)
```

Comment:
Duplicated?

Reply:
Thanks. Probably crept in due to a merge conflict.